R/mread.R

```diff
@@ -461,6 +462,7 @@ mread <- function(model, project = getOption("mrgsolve.project", getwd()),
   x@shlib[["nm_import"]] <- mread.env[["nm_import"]]
   x@shlib[["source"]] <- file.path(build[["soloc"]],build[["compfile"]])
   x@shlib[["md5"]] <- build[["md5"]]
+  x@shlib[["call_event"]] <- "EVENT" %in% names(spec)
```

Comment:
We will pass this into the C+ simulation code to tell if we are using `$EVENT` to create modeled doses.